spec/nugets_pack_spec.rb

```diff
@@ -64,6 +64,13 @@ def self.create id, curr, gen_symbols = true
   end
 end
 
+describe Config, 'when setting #nuget_gem_exe' do
```

Comment:
Ran it with ruby 2.2, but noticed now that the tests are supposed to work under 1.9.3 as well.